data/json/mapgen/helipad.json

```diff
@@ -0,0 +1,215 @@
+        ".F..W-c?w.pppppppp...|||||||||||||||||||||||..F.",
+        ".F..WmWWW.pppppppp...|%_____________C__B___|..F.",
+        ".F..pppppppppppppppppM___C_________C_CC_B__|..F.",
+        ".F..pppppppppppppppppM____________CCCCC____|..F.",
+        ".F..pppppppppppppppppM_B__C___B__B___C_B___|..F.",
+        ".F..pppppyppppyppppppM________*_________B__|..F.",
+        ".F..pppppyppppyppppppM__B___C___B__C__B____|..F.",
+        ".F..pppppyppppyppp...|____C___B____________|..F.",
+        ".F..pppppyppppypppooom___________B______%__|..F.",
+        ".F..pppppyppppyppp...||||||||m|MMMMMMMMM||||..F.",
+        ".F..pppppyppppyppp...|_rrrr|y___________%__|..F.",
+        ".F..............oo...|_____|_______________|..F.",
+        "FFF.............oo...|_rrrr|_____________RR|.FFF",
+        "FTG.WwWWwWwwWwwW++WwW|M|||||_____________sR|.GTF",
+        "FFF.W----+-----+-----m_____m___IIIIIIIII_75|.FFF",
+        ".F..W-?ddWdc--dW--WWW|||||||___IUUUUUUUI___|..F.",
+        ".F..Wy-cfWd?f?dW--W&W|_____m___IUUUUUUUI_RR|..F.",
+        ".F..WWWWWWWWWWWWy-+-W|ggggg|___IUUUUUUUI_sR|..F.",
+        ".F..W[[[HH##O#SW--WSW|||||||___IUUUUUUUI_85|..F.",
+        ".F..W----------+--WWW|lllll|___IUUUUUUUI___|..F.",
+        ".F..W-cttc---1WW--+-W|_____M___IUUUUUUUI___|..F.",
+        ".F..W-cttc-y-2W<--WhW|lllll|___IUUUUUUUI%__|..F.",
+        ".F..WWwwwWwwwWWW++WWW||||||||m|MMMMMMMMM||||..F.",
+        ".F.......3P.....oo...........o.=========6666..F.",
+        ".F..========================================..F.",
+        ".F..========================================..F.",
+        ".F..========================================..F.",
+        "FFF.========================================.FFF",
+        "FTG..o,,,,,o..o,,,,,,,,,,,,,,,,,,o..o,,,,,o..GTF",
+        "FFF..o=====o..o==================o..o=====o..FFF",
+        ".F..$=========o$==,,========,,$==o==========..F.",
+        ".F..==========o===,,========,,===o==========..F.",
+        ".F..=,=====,==o===,,========,,===o=,=====,==..F.",
+        ".F..=,=====,==o===,,========,,===o=,=====,==..F.",
+        ".F..=,=====,==o===,,,,,,,,,,,,===o=,=====,==..F.",
+        ".F..=,,,,,,,==o===,,,,,,,,,,,,===o=,,,,,,,==..F.",
+        ".F..=,,,,,,,==o===,,========,,===o=,,,,,,,==..F.",
+        ".F..=,=====,==o===,,========,,===o=,=====,==..F.",
+        ".F..=,=====,==o===,,========,,===o=,=====,==..F.",
+        ".F..=,=====,==o===,,========,,===o=,=====,==..F.",
+        ".F..==========o==================o==========..F.",
+        ".F..==========o==================o==========..F.",
+        "FFG............FGF............FGF............GFF",
+        "FTFFFFFFFFFFFFFFTFFFFFFFFFFFFFFTFFFFFFFFFFFFFFTF",
+        "FFF............FFF............FFF............FFF"
+      ],
+      "palettes": [ "helipad_palette" ],
+      "toilets": { "&": {  } },
+      "gaspumps": { "$": {  } },
+      "items": {
+        "B": {
+          "item": {
+            "subtype": "distribution",
```

Comment:
This seems like something that could be defined in an itemgroup, perhaps in supplies.json. I can see other people having use for drums of gasoline and motor oil

Reply:
I've added it as you suggested. I will give it some test about spawned amounts - I am not sure how charges work with fuels exactly. Besides I'd like to have some opinion on how much fuel spawned barrels should have. 
I also think motor oil might need to have different container than plastic jerrycan.